templates/bundles/DemosPlanProcedureBundle/DemosPlanProcedure/administration_split_statement.html.twig

```diff
@@ -7,6 +7,19 @@
         {% set submitTypeOptions = submitTypeOptions|merge([{ label: label, value: value }]) %}
     {% endfor %}
 
+    {% set addonProps = {
+        'procedureId': procedure,
+        'statementId': statementId,
+        'statementExternId': statementExternId,
+        'procedureId': procedure,
```

Comment:
```suggestion
        'procedureId': procedure,
        'statementId': statementId,
        'statementExternId': statementExternId,
```

procedureId is double

Reply:
Oh thanks for catching this, removed the duplication here [eeb1237](https://github.com/demos-europe/demosplan-core/pull/951/commits/eeb1237e917186797e6b605faac0596739f7fcc9)